examples/open_loop_control_fetch/generate_actions.py

```diff
@@ -4,179 +4,199 @@
+        df = pd.concat([df, pd.DataFrame(action)])
+
+    # lower arm_joint_1, arm_joint_3, raise arm_joint_5
+    for _ in range(25):
+        action = build_action_dict()
+        action["arm_joint_1.add_to_angular_orientation_target"] = [np.array([0.0, 0.0, -1.0], dtype=np.float64)]
+        action["arm_joint_3.add_to_angular_orientation_target"] = [np.array([0.0, 0.0, -1.0], dtype=np.float64)]
+        action["arm_joint_5.add_to_angular_orientation_target"] = [np.array([0.0, 0.0, 1.0], dtype=np.float64)]
+        action["gripper_finger_joint_l.add_force"] = [np.array([-1000.0, 0.0, 0.0], dtype=np.float64)]
+        action["gripper_finger_joint_r.add_force"] = [np.array([-1000.0, 0.0, 0.0], dtype=np.float64)]
+        df = pd.concat([df, pd.DataFrame(action)])
+
+    # lower arm_joint_1, raise arm_joint_3 and drop the object
+    for _ in range(20):
+        action = build_action_dict()
+        action["arm_joint_1.add_to_angular_orientation_target"] = [np.array([0.0, 0.0, 1.0], dtype=np.float64)]
+        action["arm_joint_3.add_to_angular_orientation_target"] = [np.array([0.0, 0.0, -1.0], dtype=np.float64)]
+        df = pd.concat([df, pd.DataFrame(action)])
+
+    # raise arm_joint_3
+    for _ in range(20):
+        action = build_action_dict()
+        action["arm_joint_3.add_to_angular_orientation_target"] = [np.array([0.0, 0.0, 1.0], dtype=np.float64)]
+        df = pd.concat([df, pd.DataFrame(action)])
+
+    # turn left
+    for _ in range(12):
+        action = build_action_dict()
+        action["wheel_joint_l.add_to_angular_velocity_target"] = [np.array([0.1, 0.0, 0.0], dtype=np.float64)]
+        action["wheel_joint_r.add_to_angular_velocity_target"] = [np.array([-0.1,  0.0, 0.0], dtype=np.float64)]
+        df = pd.concat([df, pd.DataFrame(action)])
+
+    # turn right (negates accumulated "turn left" torque values)
+    for _ in range(12):
+        action = build_action_dict()
+        action["wheel_joint_l.add_to_angular_velocity_target"] = [np.array([-0.1, 0.0, 0.0], dtype=np.float64)]
+        action["wheel_joint_r.add_to_angular_velocity_target"] = [np.array([0.1,  0.0, 0.0], dtype=np.float64)]
+        df = pd.concat([df, pd.DataFrame(action)])
+    
+    # lower arm_joint_1, arm_joint_3, raise arm_joint_5
+    for _ in range(25):
+        action = build_action_dict()
+        action["arm_joint_3.add_to_angular_orientation_target"] = [np.array([0.0, 0.0, 1.0], dtype=np.float64)]
+        action["arm_joint_5.add_to_angular_orientation_target"] = [np.array([0.0, 0.0, 1.0], dtype=np.float64)]
+        df = pd.concat([df, pd.DataFrame(action)])
 
     # save to csv
-    actions_file = os.path.realpath(os.path.join(os.path.dirname(__file__), "actions." + args.scene_id + ".csv"))
-    df.to_csv(actions_file, float_format="%.5f", mode="w", index=False, header=True)
+    actions_file = os.path.realpath(os.path.join(os.path.dirname(__file__), "actions.pkl"))
+    df.to_pickle(actions_file)
```

Comment:
Save as CSV for consistency with the rest of our examples.

Reply:
Updated code to use `csv` format.